test/api/test-xr-ray.js

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import mocha from 'mocha';
+import { assert } from 'chai';
+
+import XRRay from '../../src/api/XRRay.js';
+import DOMPointReadOnly from '../../src/lib/DOMPointReadOnly.js';
+
+describe('API - XRRay', () => {
+  it('defaults to <0,0,0> origin, <0,0,-1> direction', () => {
```

Comment:
Looking through the spec, I don't see any evidence that the direction should be <0,0,-1,0> by default (rather, I'd assume <0,0,0,0>) -- what should be the case here?